superset-frontend/src/explore/components/controls/DateFilterControl.jsx

```diff
@@ -453,66 +451,45 @@ class DateFilterControl extends React.Component {
               >
                 <div
                   className="clearfix centered"
-                  style={{ marginTop: '12px' }}
+                  style={{ marginTop: '8px', display: 'flex' }}
```

Comment:
Would love to see this in Emotion with gridUnits, but not losing sleep over it.

BTW, nice to see some LESS styles disappearing with this PR :)

Reply:
Done, here and in the rest of that file